msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/msalonly/ltw/TestCase2571345.java

```diff
@@ -0,0 +1,139 @@
+//
+//  This code is licensed under the MIT License.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files(the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+package com.microsoft.identity.client.msal.automationapp.testpass.msalonly.ltw;
+
+import com.microsoft.identity.client.msal.automationapp.R;
+import com.microsoft.identity.client.msal.automationapp.testpass.broker.AbstractMsalBrokerTest;
+import com.microsoft.identity.client.ui.automation.annotations.LTWTests;
+import com.microsoft.identity.client.ui.automation.annotations.RunOnAPI29Minus;
+import com.microsoft.identity.client.ui.automation.annotations.SupportedBrokers;
+import com.microsoft.identity.client.ui.automation.app.MsalTestApp;
+import com.microsoft.identity.client.ui.automation.app.OneAuthTestApp;
+import com.microsoft.identity.client.ui.automation.broker.BrokerCompanyPortal;
+import com.microsoft.identity.client.ui.automation.broker.BrokerLTW;
+import com.microsoft.identity.client.ui.automation.interaction.FirstPartyAppPromptHandlerParameters;
+import com.microsoft.identity.client.ui.automation.interaction.PromptParameter;
+import com.microsoft.identity.client.ui.automation.interaction.microsoftsts.MicrosoftStsPromptHandlerParameters;
+import com.microsoft.identity.labapi.utilities.client.LabQuery;
+import com.microsoft.identity.labapi.utilities.constants.TempUserType;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+// Samsung GA Coverage
+// https://identitydivision.visualstudio.com/Engineering/_workitems/edit/2571345
+@LTWTests
+@RunOnAPI29Minus
+@SupportedBrokers(brokers = {BrokerLTW.class})
+public class TestCase2571345 extends AbstractMsalBrokerTest {
+    @Test
+    public void test_2571345() throws Throwable{
+        final String username = mLabAccount.getUsername();
+        final String password = mLabAccount.getPassword();
+
+        // Install new OneAuthTestApp
+        final OneAuthTestApp oneAuthTestApp = new OneAuthTestApp();
+        oneAuthTestApp.uninstall();
```

Comment:
Why are we having to uninstall these test apps every time? Is there any test rule that is installing them already? Afaik, these would not be preinstalled. These are just pushed to the apk(not installed)

Reply:
This is still not fixed

Reply:
Hi Sowmya, you might want to view the changes from all commits like this:
![Screenshot 2023-08-29 at 14 35 11](https://github.com/AzureAD/microsoft-authentication-library-for-android/assets/51412053/36b9db7f-3360-4d8b-92ac-43b330ca7298)
![Screenshot 2023-08-29 at 14 35 23](https://github.com/AzureAD/microsoft-authentication-library-for-android/assets/51412053/d21191d4-edf9-4def-9784-ac4f3007bedf)
It's showing that this line already got removed. I don't know why but seems the screenshot above marek as "outdated" is not updating with the latest change. Can you please check from the screenshot I attched above? Thanks!